src/main/java/com/unb/tracker/repository/CourseRepository.java

```diff
@@ -16,4 +16,6 @@
     @Query(value = "SELECT * FROM course WHERE course.name LIKE CONCAT('%',:pname,'%') OR course.section LIKE CONCAT('%',:pname,'%') LIMIT 10", nativeQuery = true)
     public List<Course> findByPartialName(@Param("pname")String partialName);
 
+    public Course findById(Long id);
```

Comment:
You don't actually need this, it already exists in the form of `findOne(Long id)`

Reply:
Oh cool! Didn't know about this

Reply:
thanks @jacsmith21